### ISSUE TITLE
"Ihr Testergebnis konnte nicht geladen werden" pop up is shown 

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-ios#4180]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 6s
- iOS version: 15.4 Beta 1
- App version: 2.17.1

## Describe the bug

The app shows this error while having a PCR test registered since 3 days and the result wasn't submitted:
<img src="https://user-images.githubusercontent.com/67682506/152797145-88aeec84-74df-4a11-a18f-3fa7d9a19bba.PNG" width="300">

## Steps to reproduce the issue

1. Register a PCR test
2. Don't submit a result
3. Wait a few days until the pop up is shown

## Expected behaviour

Either a clear description what the user should do should be shown or the pop up should not be shown at all. 

## Additional context

I think this is a leftover from the fix of #2694.

---
Internal Tracking ID: [EXPOSUREAPP-11732](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11732)

**Comment by Ein-Tim** [comment]
@dsarkar Thanks! The main problem is not that the pop up is shown but that there is no clear advise what the user should do or any helpful text.

**Comment by Ein-Tim** [comment]
(But for sure if the user can't do anything with this information then this pop up should be removed.)

**Comment by Ein-Tim** [comment]
@dsarkar 

This is a PRIO bug, all texts are not correctly localized (here I wasn't connected to the internet):
<img src="https://user-images.githubusercontent.com/67682506/152851284-9fa24ac6-7a52-44e7-b138-3456ebe8000f.jpeg" width="300">

**Comment by thomasaugsten** [comment]
This detailed error message is intended to catch the rare case where the key submission is failing. 
Maybe the header of the error is misleading.
Is the error occurring on each app start?

**Comment by Ein-Tim** [comment]
@thomasaugsten 

Yes, this is shown at each start of the app. The app should log this information but never ever show this "Stacktrace" to the user. If you want to show this, please implement a "Mehr Informationen" button which reveals the information.

Because as of now, when thinking from a normal user perspective: What do I do with this error message? I click on "OK" and continue what I wanted to do. Maybe I even remove the test because I think this is broken. If the message is just "Es besteht keine Verbindung zum Internet." The user exactly knows what is going on and knows what to do. 

I strongly suggest to log this information but not show this to the user.

**Comment by thomasaugsten** [comment]
Exactly the error occurs normally only once and in this cases the logging was not active.
It also an edge case that the user opens the app everytime without internet.
We will have a look to improve the formatting

**Comment by Ein-Tim** [comment]
@thomasaugsten 

> Exactly the error occurs normally only once and in this cases the logging was not active.

Ah okay, I understand. 

> It also an edge case that the user opens the app everytime without internet.

Yes but as you can see in the OP this also happens in the OP.

**Comment by ndegendogo** [comment]
> It also an edge case that the user opens the app everytime without internet.

Actually, you should not make such assumptions about how and under what conditions users are using the app ... meanwhile we have a lot of use cases that don't need internet connectivity....

**Comment by Ein-Tim** [comment]
PR #4245

**Comment by Ein-Tim** [comment]
I will try to reproduce this with version 2.18 & report back.

**Comment by Ein-Tim** [comment]
This has been fixed in version 2.18.

Closing as fixed.

**Comment by Ein-Tim** [comment]
@dsarkar 

Yes, that's exactly what I've done.

**Comment by dsarkar** [comment]
@Ein-Tim Thanks.
